Sources/Application/Game/Player.swift

```diff
@@ -0,0 +1,22 @@
+public class Player {
+    public var name: String
+    public var order: Int
+    public var correctGuesses: [String]
+    public var incorrectGuesses: [String]
+
+    public init(name: String, order: Int, correctGuesses: [String], incorrectGuesses: [String]) {
+        self.name = name
+        self.order = order
+        self.correctGuesses = correctGuesses
+        self.incorrectGuesses = incorrectGuesses
+    }
+
+    public func appendGuess(word: String, guess: Guess) {
```

Comment:
Guess manager will no longer be handling the collections. I have the players each keeping track of their own correct and incorrect guesses